web/shared/src/typing_status.js

```diff
@@ -13,22 +14,17 @@ const TYPING_STARTED_WAIT_PERIOD = 10000; // 10s
 const TYPING_STOPPED_WAIT_PERIOD = 5000; // 5s
 
 /** Exported only for tests. */
-export const state = {};
+export let state = null;
 
 /** Exported only for tests. */
 export function initialize_state() {
-    state.current_recipient = null;
-    state.next_send_start_time = undefined;
-    state.idle_timer = undefined;
+    state = null;
```

Comment:
We can remove this helper because we can just set `state = null` here and in the tests now. The naming appears to be a bit misleading in terms of what it does now.